tests/integration/security/jwt_test.go

```diff
@@ -62,335 +61,285 @@ func TestRequestAuthentication(t *testing.T) {
 				t.Fatalf("Wait for jwt-server server failed: %v", err)
 			}
 
-			// Apply the policy.
-			namespaceTmpl := map[string]string{
-				"Namespace": ns.Name(),
-			}
-
 			callCount := 1
 			if t.Clusters().IsMulticluster() {
 				// so we can validate all clusters are hit
 				callCount = util.CallsPerCluster * len(t.Clusters())
 			}
 
-			for _, cluster := range t.Clusters() {
-				client := apps.A.Match(echo.InCluster(cluster).And(echo.Namespace(apps.Namespace1.Name())))[0]
-				dest := apps.B.Match(echo.InCluster(cluster).And(echo.Namespace(apps.Namespace1.Name())))
-				t.NewSubTest(fmt.Sprintf("From %s", cluster.StableName())).Run(func(t framework.TestContext) {
-					testCases := []authn.TestCase{
-						{
-							Name:   "valid-token-noauthz",
-							Config: "authn-only",
-							Request: connection.Checker{
-								From: client,
-								Options: echo.CallOptions{
-									Target:   dest[0],
-									PortName: "http",
-									Scheme:   scheme.HTTP,
-									Headers: map[string][]string{
-										authHeaderKey: {"Bearer " + jwt.TokenIssuer1},
-									},
-									Count: callCount,
-								},
-								DestClusters: dest.Clusters(),
-							},
-							ExpectResponseCode: response.StatusCodeOK,
-							ExpectHeaders: map[string]string{
-								authHeaderKey:    "",
-								"X-Test-Payload": payload1,
+			t.NewSubTest("").Run(func(t framework.TestContext) {
```

Comment:
can you also give this a name otherwise the test log just uses `#00` .

Reply:
Done.